docs/application/native/guides/ui/efl/efl-util.md

```diff
@@ -334,10 +334,14 @@ To generate touch input events:
        }
    ```
 
-2. After setting the input device type, call the `efl_util_input_generate_touch()` function to generate touch input events:
+2. After setting the input device type, call the `efl_util_input_generate_touch()`, or `efl_util_input_generate_touch_axis()` function to generate touch input events:
 
    ```
        ret = efl_util_input_generate_touch(inputgen, 0, EFL_UTIL_INPUT_TOUCH_BEGIN, 100, 100);
+       /*
+          Since Tizen 5.5, you can generate touch events with touch axis information:
```

Comment:
If I fix Since Tizen 5.5 to From Tizen 5.5, do I fix From Tizen 5.0 to Since Tizen 5.0 or just left it?

Reply:
https://github.com/Samsung/tizen-docs/pull/556#discussion_r258824227

@Reni-90, as you can see, and as jhyuni asked, in this page "since" and "from" are mixed.

 - Since Tizen 4.0,
 - From Tizen 5.0,
 - Since Tizen 5.5,
 
For now Tizen 5.0 is the current, but it will be an old version one day.
Tizen 5.5 will be the "current version", but it will be also an old version one day.

So in developer document, 
I think considering **current version** for "since" and "from" is meaningless and can be wasting of an effort.

**So, how about using "since" for this case?**

Reference link from Andorid doc for comparing : 
 - Search result of "since API level" https://developer.android.com/s/results/?q=since%20API%20level
 - Search result of "from api level" https://developer.android.com/s/results/?q=from%20api%20level&p=%2F

Reply:
@mijin85cho 
Ideally, for the current version we use "from" and for the older version we use "since". However, it is right that now Tizen 5.0 is the current, but it will be an old version one day.
Tizen 5.5 will be the "current version", but it will also be an old version one day.
If in the developer document, considering the current version for "since" and "from" is meaningless, then I would like to go with your suggestion. 
Thank you very much for the explanation. 

@jhyuni 
Please ignore my previous comment. To maintain consistency, just change "From Tizen 5.0" to Since Tizen 5.0".

Reply:
Thanks for the understanding, @Reni-90.